argus/recipes/cloud/windows.py

```diff
@@ -41,6 +41,19 @@ def wait_for_boot_completion(self):
         LOG.info("Waiting for first boot completion...")
         self._backend.remote_client.manager.wait_boot_completion()
 
+    def set_mtu(self, interface="ipv4", subinterface_name="Ethernet",
+                mtu_value=1400, store_type='active'):
+        set_mtu_cmd = ('netsh interface {interface_type} set subinterface '
+                       '"{name}" mtu={value} store={type}'
+                       .format(interface_type=interface, value=mtu_value,
+                               name=subinterface_name, type=store_type))
+        LOG.info("Setting the MTU.")
+        try:
+            self._backend.remote_client.run_command_with_retry(set_mtu_cmd)
+        except exceptions.ArgusTimeoutError as exc:
```

Comment:
`run_command_with_retry` will throw out an `ArgusTimeoutError` in case it failed too many times, the rest of the exceptions are caught inside the method itself, don't think other kind of exceptions can propagate since they are caught inside the method.